src/resources/help/en_US/relnotes3.1.4.html

```diff
@@ -36,6 +36,7 @@
   <li>Fixed a bug where applying Log scaling to the view of a Curve plot (or lineout curve) yielded wrong results.</li>
   <li>Fixed a bug with the variable menu not being sorted properly when the variable list had mixed-case names and variable grouping was used.</li>
   <li>Fixed a bug with the conn_cmfe expression where mapping a variable from a polyhedral mesh to a point mesh failed when the mesh was material selected.</li>
+  <li>Fixed bug (<a href="https://github.com/visit-dav/visit/issues/5067">5067</a>) causing Curve plots with cycle coloring option to change their color as timestep is advanced and a similar issue for random coloring of Mesh plots.</li>
```

Comment:
I would change the release notes after the link to:

causing Curve plots with the "Curve color" set to "Cycle" to change their color as the timestep is advanced and a similar issue with the Mesh plot when the "Mesh color" or "Opaque color" was set to "Random".